lua/orgmode/capture/init.lua

```diff
@@ -216,7 +221,8 @@ function Capture:refile_to_headline(destination_file, lines, item, headline_titl
   end
 
   if not headline then
-    return self._refile_to_end(destination_file, lines, item)
+    utils.echo_info("headline '" .. headline_title .. "' does not exist in " .. destination_file .. ". Refiling to end of file instead.")
```

Comment:
also, this is not displayed due to being overwritten by `Written to ...` message afterwards. But I do think it is valuable information in the case where users tried to specify a headline but made e.g. a typo. Perhaps we should even consider not fulfilling such a refiling action, as it probably is not what was intended by the user in the first place